src/__mocks__/connectkit.tsx

```diff
@@ -0,0 +1,11 @@
+type MockConnectKitProviderProps = {
+  children: React.ReactNode;
+};
+
+const MockConnectKitProvider: React.FC<MockConnectKitProviderProps> = ({
```

Comment:
We use `FunctionComponent` and import from `react` rather than using `React.FunctionComponent`.
```suggestion
const MockConnectKitProvider: FunctionComponent<MockConnectKitProviderProps> = ({
```

Reply:
Done